src/components/layout-item/layout-item.scss

```diff
@@ -0,0 +1,71 @@
+:host {
```

Comment:
Should we consider a margin variable or other spacing modifier like we have on input or rely on the content placed inside?

Reply:
We can include padding by default and allow a user to remove it with a variable if that sounds good?